jonas/desktop/custom.nix

```diff
@@ -0,0 +1,14 @@
+{ config, pkgs, ... }:
+
```

Comment:
```
let
  nixgl = "${nixgl.nixGLIntel}/bin/nixGLIntel";
in
```

There are probably other ways to wrap programs, but that one gives you ultimate flexibility

Reply:
Ah, makes sense